examples/pytorch/gat/gat.py

```diff
@@ -53,24 +60,34 @@ def forward(self, nodes):
                 ret = self.residual_fc(nodes.data['h']) + ret
             else:
                 ret = nodes.data['h'] + ret
-        return {'head%d' % self.headid : self.activation(ret)}
+        return {'head%d' % self.headid: self.activation(ret)}
+
 
 class GATPrepare(nn.Module):
     def __init__(self, indim, hiddendim, drop):
         super(GATPrepare, self).__init__()
-        self.fc = nn.Linear(indim, hiddendim)
-        self.drop = drop
-        self.attn_l = nn.Linear(hiddendim, 1)
-        self.attn_r = nn.Linear(hiddendim, 1)
+        self.fc = nn.Linear(indim, hiddendim, bias=False)
+        if drop:
+            self.drop = nn.Dropout(drop)
+        else:
+            self.drop = 0
+        self.attn_l = nn.Linear(hiddendim, 1, bias=False)
+        self.attn_r = nn.Linear(hiddendim, 1, bias=False)
+        nn.init.xavier_normal_(self.fc.weight.data, gain=1.414)
+        nn.init.xavier_normal_(self.attn_l.weight.data, gain=1.414)
+        nn.init.xavier_normal_(self.attn_r.weight.data, gain=1.414)
```

Comment:
I remember you changed `gain` previously... I think it's because you copy-pasted and overwrote those changes.

Please make sure other places don't have similar errors.